### ISSUE TITLE
PDFs are not loading properly 

### ISSUE THREAD

**Issue by dasaderi** [PREreview/rapid-prereview#75]
e.g. https://outbreaksci.prereview.org/10.1101/2020.01.26.20018754

**Comment by sballesteros** [comment]
- 10.31219/osf.io/abx7s seems to be a bug
- the medrxiv / biorxiv ones seem to come from medrxiv / biorxiv responding with a 503 error.

**Comment by dasaderi** [comment]
Thanks, @sballesteros. So it's not an issue on our end? What can we do?

**Comment by sballesteros** [comment]
Not sure, I will check in more details tonight after work. The URL we want to load seems to work (https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958.full.pdf) so not sure why we get the 503. 
Just a vague theory but given that we are proxying the request through https://github.com/PREreview/rapid-prereview/blob/master/src/routes/api-routes.js#L374-L398 and that the URL works without the proxy maybe biorxiv / medrxiv blacklisted us?

**Comment by sballesteros** [comment]
https://outbreaksci.prereview.org/10.3389/fpsyg.2019.01557 works and it's the exact same code path.

**Comment by georgiamoon** [comment]
We also tested and found that the arxiv is down, so not a bug for rapid-prereview from that perspective.

Instead we should probably change this bug to be that rapid needs a way to handle in the UI for when the arxiv servers are having issues and communicate that to the user.

**Comment by sballesteros** [comment]
As far as i can tell the medrxiv and biorxiv servers are not down for the DOI I checked and the issue is only when the PDF is proxied (the raw URL works fine)

**Comment by sballesteros** [comment]
it seems that medrxiv / biorxiv are now using an extra layer of protection to protect from DDoS attacks and maybe we got blacklisted as part of it?

**Comment by georgiamoon** [comment]
@sballesteros we were getting 503s here. I think they might be having an intermittent issue due to increased load. My point is that rapid-prereview probably needs a way to handle the fact that the source material might not be available, and so have a way provide a good user experience in that event.

**Comment by sballesteros** [comment]
sure assuming that it's intermittent (which I am not 100% sure yet) improving the error message is definitely a good idea and could be done in a new issue 👍

**Comment by sballesteros** [comment]
also AFAIK arxiv and biorxiv / medrxiv are hosted on different infrastructure so their status should be independent.

**Comment by sballesteros** [comment]
<img width="967" alt="Screen Shot 2020-03-18 at 4 11 55 PM" src="https://user-images.githubusercontent.com/1141327/77015830-39ab5600-6933-11ea-96f5-faf1ed3fdfd6.png">

^^ that's the new protection layer from biorxiv / medrxiv which may be creating an issue for our proxy.

**Comment by dasaderi** [comment]
Another user signaled this problem with the following preprints:

THE CAUSE OF PANIC AT THE OUTBREAK OF COVID-19 IN SOUTH AFRICA – A COMPARATIVE ANALYSIS WITH SIMILAR OUTBREAK IN CHINA AND NEW YORK
https://outbreaksci.prereview.org/10.31730/osf.io/sy54p

COVID-19 in unequally ageing European regions
https://outbreaksci.prereview.org/10.31219/osf.io/abx7s

**Comment by sballesteros** [comment]
yes any OSF preprint should be like that, i have a hint of a PR that I am hoping to send soon

**Comment by georgiamoon** [comment]
@dasaderi @sballesteros is this issue still relevant? It seems like the OSF preprint issue is different, but I don't know if there's an issue for that?

**Comment by dasaderi** [comment]
I'm not sure.

**Comment by murkatr** [comment]
From May 12th Meeting: 
- Not COVID specific
- No feedback for the user on why no data is returned and why/where
